Lib/test/test_ntpath.py

```diff
@@ -77,27 +78,6 @@ def tester(fn, wantResult):
               %(str(fn), str(wantResult), repr(gotResult)))
 
 
-def _parameterize(*parameters):
-    """Simplistic decorator to parametrize a test
-
-    Runs the decorated test multiple times in subTest, with a value from
-    'parameters' passed as an extra positional argument.
-    Calls doCleanups() after each run.
-
-    Not for general use. Intended to avoid indenting for easier backports.
-
-    See https://discuss.python.org/t/91827 for discussing generalizations.
-    """
-    def _parametrize_decorator(func):
-        def _parameterized(self, *args, **kwargs):
-            for parameter in parameters:
-                with self.subTest(parameter):
-                    func(self, *args, parameter, **kwargs)
-                self.doCleanups()
-        return _parameterized
-    return _parametrize_decorator
-
-
```

Comment:
For `test_ntpath` and `test_posixpath`, could you leave `_parameterize` in for now, making it an adapter for `support.subTests`?

This was added recently, in a security change that was backported everywhere and received limited review. I think it would be good to try keeping the tests stable (textually) for a while (a few months?), to make backports easier.